en/docs/learn/fraud-detection.md

```diff
@@ -0,0 +1,422 @@
+system intrusions. A system intrusion is an intentional attempt to access/modify unauthorized information in the
+system and convert the system to an unreliable state through information leakage or denial of service attacks.
+
+Fraud detection involves monitoring the behavior of users in order to estimate anomalous behavior. These behaviors
+can have multiple facets including fraud, intrusion, account defaulting and delinquency. At present fraud detection
+has been implemented by utilizing various methods such as data mining, statistics, and artificial intelligence. All
+of these methods analyze patterns and data about users to identify and prevent suspicious behaviors.
+
+### Risk-Based Authentication
+
+Password-based authentication is still one of the primary authentication mechanisms in most of the enterprise systems
+. It is highly insecure and is prone to phishing attacks. Therefore organizations should enforce additional security
+measures to ensure the security of their users. Multi-factor authentication is a widely accepted mechanism to enhance
+ the security of user authentication. However, it is not a popular choice among users due to the negative impact it
+ has on user experience. Risk based authentication (RBA)  is another mechanism that provides secure authentication
+ while having a minimal impact on user experience. Furthermore, it ensures protection from security risks such as
+ password database leaks, credential stuffing, insecure passwords and guessing attacks.
+
+RBA is an adaptive authentication mechanism which applies different authentication levels to a user, based on a risk
+factor associated with the user. This risk factor can be used as a deciding point when predicting the probability of
+a system compromise. As the risk increases, the process of authentication should become more strict and comprehensive.
+
+RBA systems work by analysing additional properties of user us which are contextually available during login. These
+include the device of login, geolocation, IP address etc. Based on these properties a risk score is calculated for
+the user. This risk score is classified and based on the severity level, additional authentication factors are
+requested from the user. Examples for additional authentication factors are Email/SMS based verification, prompting
+CAPTCHA and so on.
+
+### Identity Verification
+
+Presently, data breaches are a major concern when it comes to ensuring the security of a system, as it provides means
+ for identity impersonation. This allows fraudulent users to get easy-access to your system. Therefore it is
+ important to verify the **identity** of a user prior to user onboarding.
+
+**Identity** of a person refers to a set of attributes that can be used to describe a person uniquely. These include
+personal details of the user such as name, date of birth, gender, ethnicity and also it can relate to location or
```

Comment:
ethnicity, and it can also relate to location or communication-based attributes...